docs/devops-guide/docker.md

```diff
@@ -9,27 +9,33 @@ sidebar_label: Docker
 In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
 follow these steps:
 
-1. Download and extract the [latest release]
+1. Install git:
```

Comment:
This is not needed unless you are installing the development version, which is not recommended.